ui/packages/consul-ui/translations/en-us.yaml

```diff
@@ -1 +1,48 @@
 common:
+  brand:
```

Comment:
this is awesome!

Reply:
I can't remember if we discussed this. I do need to use a translation file atm for auth methods. Let's chat and get on the same page about this.

Reply:
Sounds good, ping me when you are around and we can decide how we want to do it